test/integration/default/serverspec/default_spec.rb

```diff
@@ -57,6 +57,9 @@
         '/opt/kafka/config/log4j.properties'
       end
     end
+    it 'has > 1 line' do
+      expect(File.open('/opt/kafka/config/log4j.properties', 'r').readlines.size).to be > 1
+    end
```

Comment:
I recommend you use the [file InSpec Resource](https://docs.chef.io/inspec/resources/file/) for this test instead of using raw ruby and actually check some of the content specifically.